evm-bridging/src/lib/CrossVMBridgeFulfillmentUpgradeable.sol

```diff
@@ -0,0 +1,66 @@
+// SPDX-License-Identifier: Unlicense
+pragma solidity 0.8.24;
+
+import {CrossVMBridgeCallableUpgradeable} from "./CrossVMBridgeCallableUpgradeable.sol";
+import {ERC721Upgradeable} from "@openzeppelin/contracts-upgradeable/token/ERC721/ERC721Upgradeable.sol";
+import {ERC165Upgradeable} from "@openzeppelin/contracts-upgradeable/utils/introspection/ERC165Upgradeable.sol";
+import {ICrossVMBridgeFulfillment} from "../interfaces/ICrossVMBridgeFulfillment.sol";
+
+abstract contract CrossVMBridgeFulfillmentUpgradeable is CrossVMBridgeCallableUpgradeable, ERC165Upgradeable, ERC721Upgradeable {
```

Comment:
Adapted for upgradable contracts from [flow-evm-repo draft PR](https://github.com/onflow/flow-evm-bridge/pull/168/files#diff-77f2f2408b4f1cebb1fb2f47739584dbaefe42648e5e86b08d15febc34d7ab3dR7)  - the functionality is what's currently expected to register a custom contract in upcoming upgraded bridge